docs/source/account.rst

```diff
@@ -4,87 +4,64 @@ Account
 Background
 ----------
 
-StreamAlert can accept ingest, analyze and alert on data from a wide range of environments and operating systems.
+StreamAlert can ingest, analyze and alert on data from a wide variety of environments, applications, or systems.
 
-However, the StreamAlert application itself, and its supporting services, must be deployed in an AWS account of your choosing. Deployment is automated via Terraform and can be done against an existing AWS account or a new isolated AWS account you’ve created for this purpose.
+However, the StreamAlert application itself and its supporting services must be deployed in an AWS account of your choosing, either new or existing. Deployment is also fully automated via Terraform.
 
-Configuration
--------------
-As outlined above, choose or create the AWS account you will use to house the StreamAlert infrastructure.
-
-If you are interested in simply demo'ing StreamAlert, you can create a hassle free-tier AWS account `here <https://aws.amazon.com/free/>`_.
-
-account_id
-~~~~~~~~~~
+If do not currently have an AWS account and you are interested in simply testing StreamAlert, you can create a hassle free-tier AWS account `here <https://aws.amazon.com/free/>`_.
```

Comment:
I think maybe this should be `hassle-free free tier` account but then that sounds kinda silly